data/apps.yml

```diff
@@ -169,4 +169,11 @@
   description: Internet RSS Newsfeed Reader
   stable: http://feedreader.xarbit.net/feedreader-repo/feedreader.flatpakref
 
+- name: Scribus
+  description: Open Source Page Layout and Desktop Publishing (DTP)
+  nightly: http://drjurf.tk/scribus-flatpak/scribus-nightly.flatpakref 
```

Comment:
... the entire Flathub website is plain text. Should probably move it on to flathub.org infra.